python/tests/res/enkf/test_enkf.py

```diff
@@ -211,31 +211,22 @@ def test_run_context(self):
             iactive[1] = False
             run_context = main.getRunContextENSEMPLE_EXPERIMENT( fs , iactive )
 
-            self.assertEqual( len(run_context) , 8 )
+            self.assertEqual( len(run_context) , 10 )
 
             with self.assertRaises(IndexError):
-                run_context[8]
+                run_context[10]
 
             with self.assertRaises(TypeError):
                 run_context["String"]
 
-            run_arg = run_context[0]
+            self.assertIsNone( run_context[0] )
+            run_arg = run_context[2]
             self.assertTrue( isinstance( run_arg , RunArg ))
 
-            with self.assertRaises(ValueError):
-                run_context.iensGet(0)
-
-
-            with self.assertRaises(ValueError):
-                run_context.iensGet(1)
-
-            arg0 = run_context[0]
-            arg2 = run_context.iensGet( 2 )
-            #self.assertEqual( arg0 , arg2 )
 
 
     def test_run_context_from_external_folder(self):
-        with TestAreaContext('enkf_test') as work_area:
+        with TestAreaContext('enkf_test', store_area = True) as work_area:
```

Comment:
Should `store_area` be defaulted again perhaps?

Reply:
Yes